examples/time-of-day/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: time-of-day-check.0.0.1
+  annotations:
+    argo-hub/version: '0.0.1'
+    argo-hub/description: 'An example of a workflow template that uses the time of day to limit if a pipeline continues or not'
+    argo-hub/categories: 'argo'
+    argo-hub/license: 'MIT'
+    argo-hub/owner_name: "Paul O'Reilly"
+    argo-hub/owner_email: "paul.oreilly@codefresh.io"
+    argo-hub/owner_avatar: 'https://github.,com/paul-codefresh.png'
+    argo-hub/owner_url: 'https://github.com/paul-codefresh'
+    argo-hub/icon_url: 'https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/time-of-day-check/assets/icon.svg'
+    argo-hub/icon_background: "#f4f4f4"
+spec:
+  serviceAccountName: codefresh-sa
+  entrypoint: example-workflow
+  arguments:
+    parameters:
+      - name: TIME_WINDOW_STARTS
+        default: '09:00'
+      - name: TIME_WINDOW_FINISHES
+        default: '16:00'
+      - name: TIME_WINDOW_INVERTED
+        default: 'false'
+      - name: TIME_WINDOW_TIMEZONE
+        default: 'UTC'
+      - name: TIME_WINDOW_DAYS
+        default: 'Monday Tuesday Wednesday Friday'
+  templates:
+  - name: time-of-day-check-template
+    metadata:
+      annotations:
+        argo-hub-template/description: 'Checks the day and time and fails if the current time is outside of a specified window or day of the week'
+        argo-hub-template/icon_url: 'https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/time-of-day-check/assets/icon.svg'
```

Comment:
`examples/time-of-day-check/assets` should be `examples/time-of-day/assets to match folder name